plugin/src/functionalTest/kotlin/io/qalipsis/gradle/cloud/QalipsisCloudPluginFunctionalTest.kt

```diff
@@ -0,0 +1,861 @@
+package io.qalipsis.gradle.cloud
+
+import io.kotest.matchers.string.shouldContain
+import io.qalipsis.gradle.MockserverContainer
+import io.qalipsis.gradle.SSETestServer
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.AfterEach
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.io.TempDir
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import org.junit.jupiter.params.provider.ValueSource
+import org.mockserver.client.MockServerClient
+import org.mockserver.model.Header
+import org.mockserver.model.HttpRequest.request
+import org.mockserver.model.HttpResponse.response
+import org.testcontainers.junit.jupiter.Testcontainers
+import java.nio.file.Path
+import java.time.Instant
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.time.temporal.ChronoUnit
+import java.util.stream.Stream
+import kotlin.io.path.writeText
+
+@Testcontainers
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+internal open class QalipsisCloudPluginFunctionalTest : MockserverContainer() {
```

Comment:
Split this test class into several ones (one for each task, one for the plugin application in order to verify that the expected tasks are automatically created)